src/io.cpp

```diff
@@ -453,26 +399,22 @@ GeneratedPipelineInput::GeneratedPipelineInput(const Catalog &catalog,
     }  
 }
 
-Quaternion PromptSphericalAttitude() {
-    float ra = Prompt<float>("Boresight right ascension");
-    float dec = Prompt<float>("Boresight declination");
-    float roll = Prompt<float>("Boresight roll");
-    return SphericalToQuaternion(DegToRad(ra), DegToRad(dec), DegToRad(roll));
-}
-
-PipelineInputList PromptGeneratedPipelineInput() {
+PipelineInputList GetGeneratedPipelineInput(const PipelineOptions &values) {
     // TODO: prompt for attitude, imagewidth, etc and then construct a GeneratedPipelineInput
-    int numImages = Prompt<int>("Number of images to generate");
-    int xResolution = Prompt<int>("Horizontal Resolution");
-    int yResolution = Prompt<int>("Vertical Resolution");
-    float xFovDeg = Prompt<float>("Horizontal FOV (in degrees)");
+    int numImages = values.generate;
+    int xResolution = values.horizontalRes;
+    int yResolution = values.verticalRes;
+    float xFovDeg = values.fov;
```

Comment:
As described in other comment, should use pixel size and focal length if provided, or fall back to fov if pixel size and focal length are not specified.